tests/testthat/test-bootstrap.R

```diff
@@ -58,6 +58,21 @@ test_that("bootstrap succeeds with empty repos", {
 
 })
 
+test_that("bootstrap succeeds with fallback URL", {
+
+  skip_on_cran()
+  skip_on_os("windows")
+
+  renv_tests_scope()
+  renv_scope_options(repos = c(CRAN = "https://broken_url"))
+
+  library <- renv_libpaths_default()
+  bootstrap(version = "1.0.0", library = library)
```

Comment:
Version="1.0.0" is a local version and should be changed to a cran available version like 0.12.3 to actually download from cran. But that seems to be not possible during the tests?

Reply:
Any hints? Thanks

Reply:
`renv` sets up its own local CRAN-like repository with "fake" versions of packages available, and those should be used during testing.

(Relying on CRAN itself means tests will fail intermittently if internet connectivity is spotty on the CRAN machines, which we do not want)

Reply:
Ok, will then remove the test case, as the changed code is not testable that way.

Anything else to change/do? Any comment to the actual changed code? Thanks